04-job-dsl-for-github-org/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM jenkins/jenkins:2.71-alpine
 
 ENV JENKINS_REF /usr/share/jenkins/ref
+RUN mkdir -p $JENKINS_REF
 
 # install jenkins plugins
-COPY jenkins-home/plugins.txt $JENKINS_REF
+COPY jenkins-home/plugins.txt $JENKINS_REF/
 RUN /usr/local/bin/plugins.sh $JENKINS_REF/plugins.txt
```

Comment:
ok, instead of merging this PR, I pushed a fix for all the examples, if you rebase to master it will be good. aed632be62fa231354dea7f54aad007ea210b62f 
Thanks!